components/chart/trendlines.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Trendlines
+page_title: Chart Trendlines
+description: Trendlines in the Charts for Blazor.
```

Comment:
The [meta description](https://docs.telerik.com/style-guide/metadata#guidelines) must be at least 100 symbols long. The optimal length is 150. Try building one or two sentences that serve as a Call-to-Action (CTA).